lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -38,30 +39,41 @@ def convert_limits(lim, converter):
         # Cast to builtin floats as they have nicer reprs.
         return map(float, lim)
 
-    xconverter = axes.xaxis.converter
-    xmin, xmax = convert_limits(axes.get_xlim(), xconverter)
-    yconverter = axes.yaxis.converter
-    ymin, ymax = convert_limits(axes.get_ylim(), yconverter)
-    general = [('Title', axes.get_title()),
-               sep,
-               (None, "<b>X-Axis</b>"),
-               ('Left', xmin), ('Right', xmax),
-               ('Label', axes.get_xlabel()),
-               ('Scale', [axes.get_xscale(),
-                          'linear', 'log', 'symlog', 'logit']),
-               sep,
-               (None, "<b>Y-Axis</b>"),
-               ('Bottom', ymin), ('Top', ymax),
-               ('Label', axes.get_ylabel()),
-               ('Scale', [axes.get_yscale(),
-                          'linear', 'log', 'symlog', 'logit']),
-               sep,
-               ('(Re-)Generate automatic legend', False),
-               ]
+    axis_map = axes._axis_map
+    axis_converter = {
+        axis: getattr(getattr(axes, f'{axis}axis'), 'converter')
```

Comment:
And/or use the value from axis_map here (I am quite sure that `getattr(axes, f'{axis}axis')` = `axis_map[axis]` (so iterate over the dict).

Reply:
One can use `getattr(axes, f'{axis}axis').converter` here (in any combination of the above, not need to `getattr` for that.

Reply:
Understood. In that case, I'll change those loops to iterate over the dict so as to reduce all the `getattr` calls. I wasn't sure about the equality. Thanks!